tests/product.test.js

```diff
@@ -0,0 +1,144 @@
+const request = require('supertest')
+const app = require('../app')
+const { Product } = require('../models')
+const { generate } = require('../helpers/jwt')
+
+let user = {
+	email: 'admin@gmail.com',
+	password: 'pass1234',
+	role: 'admin',
+}
+
+let access_token = generate(user)
+
+beforeAll(() => {
+	Product.truncate({ restartIdentity: true })
+		.then((res) => {
+			console.log('reset')
+		})
+		.catch((err) => {
+			console.log(err)
+		})
+})
+
+describe('POST /', () => {
+	describe('Success 201, Product created', () => {
+		it(' should success and create new object of product', async () => {
+			let product = {
+				name: 'ps5',
+				image_url: 'https://www.google.com/',
+				price: 150000,
+				stock: 10,
+			}
+
+			const res = await request(app)
+				.post('/products')
+				.send(product)
+				.set('access_token', access_token)
+			expect(res.status).toBe(201)
+			expect(typeof res.body).toEqual('object')
+			expect(res.body).not.toStrictEqual({
```

Comment:
jangan pakai yg not ini karena kita sudah tau isi product kita 
name: 'ps5',
				image_url: 'https://www.google.com/',
				price: 150000,
				stock: 10,
lebih baik kita expect satu2, misal expect(res.body).toHaveProperty('stock', 10), dst